### ISSUE TITLE
Language dropdown

### ISSUE THREAD

**Issue by samyue888** [ocupop/librestream#146]
Please see the comps for the language dropdown here:
https://www.dropbox.com/s/90tdxrm8l60wu6s/2827-LIB_website-2021_Language%20Selector.pdf?dl=0

Each language will link to the homepage which we will input. Please allow this to be editable in WP so that our team and add/remove items.

Please keep this on a staging site until the multilingual sites are ready to go live. Thanks!

**Comment by samyue888** [comment]
@paulstroot Could we set a deadline of mid-August for this? Thanks

**Comment by paulstroot** [comment]
Yes. I think that will be do-able. Am I correct in understanding that each of the links in the drop-down will just link to a different external URL? It's no real clear to me since the design shows radio buttons next to the language selections.

**Comment by samyue888** [comment]
Yes, each language would link to an external URL (open in the same tab). Please let me know if you have other questions, thanks!

**Comment by paulstroot** [comment]
Work in progress can be found here:
librestreamlanguagedropdown.gatsbyjs.io

I have not added those radio buttons yet, and personally think it makes the interface a little confusing because the click is a link, and not really a radio selection, is it okay if we omit that?

I also made the drop-down menu appear under the label, instead of over top of it, so that there can be a click-to-close action. Right now clicking a second time could inadvertently select one of the languages in the menu.

The languages can be set in the CMS admin under Global Settings. if no languages are activated the language drop-down will not appear on the site.

**Comment by samyue888** [comment]
Thanks @paulstroot Looks great so far! The Japanese one seems to be indented a bit?

I agree with the radio buttons and have just sent an email to Chuck to confirm omittance.

**Comment by paulstroot** [comment]
The Japanese indent that you are seeing is just that character in the font. Nothing is actually indented.

**Comment by samyue888** [comment]
Noted. There also seems to be an extra gap at the bottom. Could we remove that?
![image](https://user-images.githubusercontent.com/81642726/127697421-bc33658a-1817-4d41-8ede-64d7d8faee60.png)

**Comment by paulstroot** [comment]
@samyue888 - that extra space has been removed

By the way, we do not need to wait until the other languages are ready to push this into the main branch. The language dropdown will not appear if we have not activated any languages in the CMS, so we can just hide it until it's ready.

**Comment by samyue888** [comment]
Noted. How do we de/activate it? Please note that the links we add to the buttons would be external sites. There would be no additional languages in the current CMS.

**Comment by paulstroot** [comment]
@samyue888 - The languages can be set in the CMS admin under Global Settings. if no languages are activated the language drop-down will not appear on the site.
